setup.py

```diff
@@ -85,16 +85,8 @@ def install_needed_req(import_name, package_name=None, min_version=None, max_ver
 
 requirements = common_requirements + ['qiskit-terra>=0.19.1', 'scipy>=1.0']
 
-if not hasattr(setuptools,
-               'find_namespace_packages') or not inspect.ismethod(
-                    setuptools.find_namespace_packages):
-    print("Your setuptools version:'{}' does not support PEP 420 "
-          "(find_namespace_packages). Upgrade it to version >='40.1.0' and "
-          "repeat install.".format(setuptools.__version__))
-    sys.exit(1)
-
 VERSION_PATH = os.path.join(os.path.dirname(__file__),
-                            "qiskit", "providers", "aer", "VERSION.txt")
+                            "qiskit_aer", "VERSION.txt")
```

Comment:
Do we want to pair this with a version bump in the required version of Terra?  The current requirement is 0.19.1.

Reply:
Yeah, I can. We'll need to wait until terra 0.21.0 to do this since we'll need https://github.com/Qiskit/qiskit-terra/pull/5089 with this.

Reply:
I think I just made a PR that increased the version requirement to 0.20 with a release note, so that'll want updating if we make this change.